src/module-2/bootstrap-design/index.html

```diff
@@ -0,0 +1,351 @@
+               </div>
+               <div class="form-check">
+                  <input class="form-check-input" type="checkbox" value="" id="flexCheckDefault">
+                  <label class="form-check-label d-flex justify-content-between  " for="flexCheckDefault">
+                     <p class="mb-0">Sony </p>
+                     <p class="mb-0">1920</p>
+                  </label>
+               </div>
+               <div class="form-check">
+                  <input class="form-check-input" type="checkbox" value="" id="flexCheckDefault">
+                  <label class="form-check-label d-flex justify-content-between  " for="flexCheckDefault">
+                     <p class="mb-0">Metra </p>
+                     <p class="mb-0">1920</p>
+                  </label>
+               </div>
+               <div class="form-check">
+                  <input class="form-check-input" type="checkbox" value="" id="flexCheckDefault">
+                  <label class="form-check-label d-flex justify-content-between  " for="flexCheckDefault">
+                     <p class="mb-0">Dyne </p>
+                     <p class="mb-0">1920</p>
+                  </label>
+               </div>
+               <div class="form-check">
+                  <input class="form-check-input" type="checkbox" value="" id="flexCheckDefault">
+                  <label class="form-check-label d-flex justify-content-between  " for="flexCheckDefault">
+                     <p class="mb-0">LG </p>
+                     <p class="mb-0">1920</p>
+                  </label>
+               </div>
+               <div class="form-check">
+                  <input class="form-check-input" type="checkbox" value="" id="flexCheckDefault">
+                  <label class="form-check-label d-flex justify-content-between  " for="flexCheckDefault">
+                     <p class="mb-0">Canon </p>
+                     <p class="mb-0">1920</p>
+                  </label>
+               </div>
+            </div>
+            <button type="button" class="btn btn-primary shadow  rounded mt-3 col-xl-12">CLEAR ALL FILTERS</button>
+         </div>
+         <div class="col-xl-8 pt-1 ">
+            <div class="d-flex justify-content-between align-items-baseline pb-2 ">
+               <p>7,618 results found</p>
+               <div>
+                  <button type="button" class="btn btn-primary shadow  rounded  pt-0  ps-2 pe-2 ">
+                     <img src="img/Path (2).svg" alt=""> </button>
+               </div>
+            </div>
+            <div class="input-group mb-3 shadow  rounded">
+               <input type="text" class="form-control " placeholder="Recipient's username"
+                  aria-label="Recipient's username" aria-describedby="button-addon2">
+               <button class="btn btn-link btn-outline-secondary  text-decoration-none text-dark" type="button"
+                  id="button-addon2"><img src="img/search.svg" alt=""></button>
```

Comment:
strange formatting here, keep all attributes in a single line or have every from new line
```
<button class="btn btn-link btn-outline-secondary  text-decoration-none text-dark" type="button" id="button-addon2"> 
     <img src="img/search.svg" alt="">
</button>
```